### DISCUSSION TITLE
Universal scene description and Nvidia Omniverse support

### DISCUSSION THREAD

**Discussion by cat768** [o3de/o3de#8717] [Ideas]
Would USD support be implemented in the future? Being able to work with blender via Nvidia Omniverse in real time would greatly speed up my workflow. Thanks in advance.

**Comment by santorac** [comment]
The asset processing system for scene data uses AssImp which I don't believe supports USD yet. (See https://github.com/assimp/assimp/issues/1332). There has been talk about adding USD support, but I'm not sure that status of that conversation, @HogJonny-AMZN or @lmbr-pip would know more.

**Comment by DarklinuxFr** [comment]
a Blender -> OE3D connector would be relevant, right?

**Comment by lmbr-pip** [comment]
There have been some discussions around USD support but no committed work from any O3DE partner at this time. As @santorac pointed out we use AssImp for the starting point for most asset importing. However to get the most out of USD we probably need to look at natively using USD as the scene description in-engine, rather than just converting from it.

If you are interested in discussing further / proposing work in this are then SIG-content is the right SIG to engage with.
